rust/ffi/src/driver_exporter.rs

```diff
@@ -207,6 +208,16 @@ macro_rules! export_driver {
             }
             adbc_core::constants::ADBC_STATUS_OK
         }
+
+        #[allow(non_snake_case)]
+        #[no_mangle]
+        pub unsafe extern "C" fn AdbcDriverInit(
```

Comment:
why this change (L212:220)?

Reply:
As a backstop so that the entrypoint can be found. Otherwise you have to declare it manually which is a bit inconvenient.